modules/katamari/src/main/ts/ephox/katamari/api/Throttler.ts

```diff
@@ -10,19 +12,20 @@ export const adaptable = <A extends any[]>(fn: (...a: A) => void, rate: number):
   let timer: number | null = null;
   let args: A | null = null;
   const cancel = () => {
-    if (timer !== null) {
+    if (!Type.isNull(timer)) {
       clearTimeout(timer);
       timer = null;
       args = null;
     }
   };
   const throttle = (...newArgs: A) => {
     args = newArgs;
-    if (timer === null) {
+    if (Type.isNull(timer)) {
       timer = setTimeout(() => {
-        fn.apply(null, args as A);
```

Comment:
This was the bug fix, as if the throttle was called inside `fn` then `timer` would not be nullable yet and as such wouldn't throttle.